src/main/java/bit/javaoop/Company.java

```diff
@@ -0,0 +1,28 @@
+package bit.javaoop;
+
+
+public class Company {
+
+    private CEO ceo;
+    private String name;
+
+    public Company(String name, CEO ceo) {
+        this.ceo = ceo;
+        this.name = name;
+    }
+
+    public void hire(AbstractEmployee employee) {
+        ceo.addEmployee(employee);
+
+    }
+
+    @Override
+    public String toString() {
+        String result = new String();
+        result += ceo.getName() + " - CEO\n";
+        for (AbstractEmployee e : ceo.getListOfEmployees()) {
+            result += e.toString(1);
```

Comment:
Tutaj kompilator nie jest w stanie ogarnąć żeby pod spodem podmienić konkatenacje na stringBuildera bo mamy ją w pętli ale za to nawet sam podpowiada jak najedziemy na podświetlenie żeby zamienić Stringa na StringBuildera:
![image](https://user-images.githubusercontent.com/18233796/38460792-7c466128-3ac1-11e8-8b3c-c63ef014f49e.png)
wystarczy mu sieknąć alt + enter i mamy:
![image](https://user-images.githubusercontent.com/18233796/38460795-8e852126-3ac1-11e8-92cf-093595e17f83.png)